test/integration/arbitrum-mainnet/Loans.fork.t.sol

```diff
@@ -168,7 +187,7 @@ contract LoansForkTest is LoansTestBase {
         assertEq(pair.cashAsset.balanceOf(feeRecipient) - feeRecipientBalanceBefore, expectedFee);
         checkLoanAmount(loanAmount);
         skip(pair.durations[0]);
-        closeAndCheckLoan(loanId, loanAmount, 0);
+        closeAndCheckLoan(loanId, loanAmount, loanAmount, 0);
```

Comment:
Here and in other places `totalAmountToSwap` is more than `loanAmount`, since should also include the position's withdrawal.